Library/Loop.cs

```diff
@@ -38,10 +41,10 @@ public Loop(Vertex v, Edge e, Face f)
             SetFace(f);
         }
 
-        /**
-         * Insert the loop in the linked list of the face.
-         * (Used in constructor)
-         */
+        /// <summary>
+        /// Insert the <see cref="Loop"/> in to the linked list of the specified <see cref="Face"/>.
```

Comment:
The note "Used in constructor" meant that this was likely not supposed to be used by external code, but exposed publicly anyway in case people would like to do weird things.

Reply:
I kind of feel like it just shouldn't be public then. If there is something that it enables the user to do, then maybe it should have a proper API, but otherwise I think it should be private since I don't think we should let the user explicitly break things like this.